Makefile

```diff
@@ -122,16 +116,17 @@ clean: info down
 	@for i in $(DIRS); do if [ -d "web/$$i" ]; then echo "Removing web/$$i..."; docker run --rm -v $(shell pwd):/mnt $(IMAGE_PHP) sh -c "rm -rf /mnt/web/$$i"; fi; done
 
 net:
-ifeq ($(shell docker network ls -q -f Name=$(COMPOSE_NET_NAME)),)
+ifeq ($(strip $(shell docker network ls | grep $(COMPOSE_NET_NAME))),)
+ifneq ($(COMPOSE_SUBNET_RANGE),)
+	docker network create --subnet $(COMPOSE_SUBNET_RANGE) $(COMPOSE_NET_NAME)
```

Comment:
do not pass `--subnet` here - docker should give you new range when net will be created, you just need populate IPRANGE with it in .env file

Reply:
we should not run `docker network create` when we already have IPRANGE